source/elements/oneMKL/source/domains/rng/device_routines/device-rng-bernoulli.rst

```diff
@@ -0,0 +1,184 @@
+.. SPDX-FileCopyrightText: 2023 Intel Corporation
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+.. _onemkl_device_rng_bernoulli:
+
+bernoulli
+=========
+
+Generates Bernoulli distributed random values.
+
+.. rubric:: Description
+
+The ``bernoulli`` class object is used in the ``generate`` and ``generate_single`` functions
+to provide Bernoulli distributed random numbers with probability ``p`` of a single trial success,
+where :math:`p \in R; 0 \leq p \leq 1`.
+
+A variate is called Bernoulli distributed if after a trial it is equal to 1 with probability of success ``p``
+and to ``0`` with probability ``1 - p``.
```

Comment:
Sorry, I didn't understand the sentence

Reply:
Agreed, a strange one indeed. I think it's ok to remove this one as distribution description is unnecessary here